features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -413,6 +413,7 @@
   "event.default_listener.governance_identity_store.priority": "97",
   "event.default_listener.governance_identity_store.enable": true,
   "event.default_listener.governance_identity_store.data_store": "org.wso2.carbon.identity.governance.store.JDBCIdentityDataStore",
+  "event.default_listener.governance_identity_store.enable_hybrid_data_store": true,
```

Comment:
Do we need to enable this by default?

Reply:
Yes, related to the PR https://github.com/wso2-extensions/identity-governance/pull/681